src/textures/parsers/SpriteSheet.js

```diff
@@ -58,6 +58,11 @@ var SpriteSheet = function (texture, sourceIndex, x, y, width, height, config)
     var column = Math.floor((height - margin + spacing) / (frameHeight + spacing));
     var total = row * column;
 
+    if(total === 0)
```

Comment:
Can you add a space here please (to keep it consistent with the rest of the codebase)